api/android/api/src/main/java/org/nnsuite/nnstreamer/NNStreamer.java

```diff
@@ -133,11 +133,11 @@ public static boolean initialize(Context context) {
         try {
             System.loadLibrary("nnstreamer-native");
             GStreamer.init(context);
-            return nativeInitialize(context);
         } catch (Exception e) {
             e.printStackTrace();
-            return false;
```

Comment:
Is it okay to keep the progress when errors occur?

Reply:
internally checks gst_is_initialized() when called nativeInitialize

Reply:
Oh, Okay